### ISSUE TITLE
Remove ethereumjs-wallet dep

### ISSUE THREAD

**Issue by kumavis** [MetaMask/eth-simple-keyring#59]
Dropping `ethereumjs-wallet` will remove the `scrypt` dependency that uses python to try to install some things, making it hard to install on windows 😿 

can replace
https://github.com/MetaMask/eth-hd-keyring/blob/d812b9cbecd2a8b56c1edf801438a9c5ae26b922/index.js#L130-L131
with
```js
const HDKey = require('hdkey')
HDKey.fromMasterSeed(seedBuffer).derive(path)
```

`ethereumjs-wallet` is just wrapping that
https://github.com/ethereumjs/ethereumjs-wallet/blob/master/hdkey.js#L35-L37

**Comment by whymarrh** [comment]
The `ethereumjs-wallet` dependency in `eth-hd-keyring` is introduced by way of this package, `eth-simple-keyring`.

**Comment by Gudahtt** [comment]
Closed by #88
